packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -44,13 +44,18 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, ButtonVa
         borderStyle: 'solid',
         borderColor: v.borderColor,
         boxShadow: v.boxShadow,
+        transition: 'all .133s cubic-bezier(.78, 0, .22, 1)',
 
         ':hover': {
           color: v.colorHover,
           backgroundColor: v.backgroundColorHover,
           borderColor: v.borderColorHover,
         },
 
+        ':active': {
+          transform: 'scale(.9)',
```

Comment:
it would also be great to just reference an animation from `scale.ts` if possible